skyfield/jpllib.py

```diff
@@ -204,8 +206,21 @@ def __repr__(self):
 
 class ChebyshevPosition(SPICESegment):
     def _at(self, t):
-        position, velocity = self.spk_segment.compute_and_differentiate(t.tdb)
-        return position / AU_KM, velocity / AU_KM, None, None
+        try:
+            position, velocity = self.spk_segment.compute_and_differentiate(t.tdb)
+            return position / AU_KM, velocity / AU_KM, None, None
```

Comment:
It's a general principle in programming to move as many lines outside of each `try:` as possible to keep software as easy to reason about as possible, so let's move this `return` down outside the `try…except`. That way it's visually more obvious to later readers that there's nothing that could go wrong with the division that the `except` is trying to catch.